llvm/test/MC/AsmParser/labels.s

```diff
@@ -29,7 +29,7 @@ foo:
 // CHECK: .long 11
         .long "a 0"
 
-// XXCHCK: .section "a 1,a 2"
+// CHECK: .section "a 1,a 2"
 //.section "a 1", "a 2"
```

Comment:
This one looks incorrect, given that the .section directive is commented out.

Reply:
Should i uncomment .section or?

Reply:
Try un-commenting, but if that still fails, make this a COM: CHECK: